xbmc/GUIPassword.h

```diff
@@ -49,11 +62,18 @@ class CGUIPassword : public ISettingCallback
   void LockSources(bool lock);
   void RemoveSourceLocks();
   bool IsDatabasePathUnlocked(const std::string& strPath, VECSOURCES& vecSources);
+  /* \brief Tests if the user is allowed to access the path by looking up the matching Mediasource
+   \param strPath The folder path to access
+   \param strType The type of share being accessed, e.g. "music", "video", etc. See CSettings::UpdateSources()
+   \return If access is granted, returns \e true
+   */
+  bool IsMediaPathUnlocked(const std::string &strPath, const std::string &strType);
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types